app/services/omdb_service.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class OmdbService
+  include HTTParty
+  base_uri 'www.omdbapi.com'
+
+  def initialize(title, year)
+    @options = { query: { t: title, y: year, apikey: Rails.application.credentials.dig(:omdb, :apikey) } }
```

Comment:
I am not sure if this is the right place to build a query

Reply:
I've followed first example form https://github.com/jnunemaker/httparty